### DISCUSSION TITLE
Firestore App Check support

### DISCUSSION THREAD

**Discussion by raulgordejev** [invertase/react-native-firebase#5916] [Ideas] [answered]
https://firebase.google.com/docs/app-check

Firebase App Check now supports Firestore on iOS and Android. Would be awesome if the react native wrapper would also support this!

**Comment by mikehardy** [comment, marked as answer]
Hi there! We already do!

From your link:

> When you enable App Check for a service and include the client SDK in your app,

So, if you include `@react-native-firebase/app-check`, *and* you go to your firebase console and toggle on appcheck for firestore (non-enforcing to start, right?) you should start seeing verified traffic.

Please note that using AppCheck in debug or CI environment currently is quite a pain, that's disccussed elsewhere in this repo - it is possible but not great (yet): https://github.com/invertase/react-native-firebase/discussions/5660
